### ISSUE TITLE
Can we finetune on HMDB51 dataset from the mobilenetv2 pretrained weights?

### ISSUE THREAD

**Issue by malarsaravanan1991** [mit-han-lab/temporal-shift-module#32]
Hi,

Thanks for sharing the mobilenetV2 pretrained weights for online TSM on kinetics. 
Can we fine-tune it on HMDB51 dataset the same way mentioned in the GitHub repository? If so what might be the expected accuracy on the smaller data sets?

**Comment by tonylins** [comment]
Sure, but I have not tried how well it works using the current parameters. You can give it a shot to try out the results!

**Comment by malarsaravanan1991** [comment]
Thank you again.. I was able to get 66% accuracy.
